src/components/bcros/Address.vue

```diff
@@ -6,7 +6,13 @@
         <div class="text-gray-900 pb-1">
           {{ $t('label.address.addressType.delivery') }}
         </div>
-        <BcrosAddressDisplay :address="address.deliveryAddress" />
+        <BcrosAddressDisplay
+          v-if="address?.deliveryAddress"
+          :address="address.deliveryAddress"
+        />
+        <div v-else>
+          {{ $t('text.general.notEntered') }}
+        </div>
```

Comment:
If delivery address is null, show (Not entered).

Example: https://business-dashboard-dev--pr-183-7cbof13k.web.app/FM1130015